src/load.jl

```diff
@@ -1,85 +1,71 @@
 function bids_layout(bidsPath::AbstractString;
-    derivative::Bool=true,
+    derivatives::Bool=true,
     specificFolder::Union{Nothing,AbstractString}=nothing,
     excludeFolder::Union{Nothing,AbstractString}=nothing,
     ses::Union{Nothing,AbstractString}=nothing,
     task::Union{Nothing,AbstractString}=nothing,
     run::Union{Nothing,AbstractString}=nothing)
 
     # Any files with these endings will be returned
-    file_pattern = ["eeg", "set", "fif", "vhdr", "edf"]
-    nPattern = 2
+    file_ending = [".eeg", ".set", ".fif", ".vhdr", ".edf"]
 
+    file_pattern = [""]
     # Extend file pattern
     if ses !== nothing
         file_pattern = push!(file_pattern, "ses-" * ses)
-        nPattern += 1
     end
 
     if task !== nothing
         file_pattern = push!(file_pattern, "task-" * task)
-        nPattern += 1
     end
 
     if run !== nothing
         file_pattern = push!(file_pattern, "run-" * run)
-        nPattern += 1
     end
 
-    # Choose a specific folder in either ./ or ./derivatives
-    if derivative && specificFolder !== nothing
-        sPath = joinpath(bidsPath, "derivatives", specificFolder)
-        #@show sPath
-    elseif specificFolder !== nothing
-        sPath = joinpath(bidsPath, specificFolder)
-        #@show sPath
-    end
-
-    # Exclude these folders when using raw data
-    if derivative && excludeFolder !== nothing
-        exclude = excludeFolder
-    elseif !derivative && excludeFolder !== nothing
-        exclude = ["derivatives", excludeFolder]
-    elseif !derivative
-        exclude = "derivatives"
+    # Choose what to ignore and check if derivatives should be used
+    exclude = [] 
+    if derivatives
+        # TODO: Bug in loading only derivatives
```

Comment:
what bug?

Reply:
Actually already fixed the bug, just forgot to delete the TODO...